packages/gatsby-theme-api-docs/src/components/resource/method/parameters.js

```diff
@@ -69,25 +74,26 @@ function ParameterRow(props) {
     isParameter: true,
   });
   return (
-    <tr key={props.parameter.name}>
+    <tr
+      key={props.parameter.name}
+      className="api-type-property" /* class name for the search crawler */
+    >
       <td>
-        <PropertyName className="name-type">
+        <PropertyName>
           <SpacingsInline scale="xs">
             <Markdown.InlineCode>{props.parameter.name}</Markdown.InlineCode>
             {props.parameter.required && <Required />}
           </SpacingsInline>
         </PropertyName>
-        <PropertyType className="name name-type">
-          {typeToRender.displayPrefix && (
-            <span className="name">{typeToRender.displayPrefix}</span>
-          )}
+        {' ' /* whitespace for the search crawler */}
```

Comment:
Nit: wrap this into a `<span>`?

```suggestion
        <span>{' ' /* whitespace for the search crawler */}</span>
```

Reply:
Why would I want that overhead?  HTML is cleanly specified how to treat whitespace and I don't need more than the effect of one piece of whitespace.